scripts/run_tests.sh

```diff
@@ -8,9 +8,10 @@ function usage() {
     exit 2
 }
 
-set -o pipefail
+#set -o pipefail
```

Comment:
and keep this too.

We want to have hard failure if any issue occurs. We can still do the error handling like this.
```
if ! command ; then
    # error handling
fi
```